cpp/tests/transform/row_bit_count_test.cu

```diff
@@ -192,6 +197,66 @@ TEST_F(RowBitCount, StringsWithNulls)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected, *result);
 }
 
+namespace {
+
+/**
+ * @brief __device__ functor to multiply input by 2, defined out of line because __device__ lambdas
+ * cannot be defined in a TEST_F().
+ */
+struct times_2 {
+  int32_t __device__ operator()(int32_t i) const { return i * 2; }
+};
+
+}  // namespace
+
+TEST_F(RowBitCount, StructsWithLists_RowsExceedingASingleBlock)
+{
+  // Tests that `row_bit_count()` can handle struct<list<int32_t>> with more
+  // than max_block_size (256) rows.
+  // With a large number of rows, computation spills to multiple thread-blocks,
+  // thus exercising the branch-stack comptutation.
+  // The contents of the input column aren't as pertinent to this test as the
+  // column size. For what it's worth, it looks as follows:
+  //   [ struct({0,1}), struct({2,3}), struct({4,5}), ... ]
+
+  using namespace cudf;
+  auto constexpr num_rows = 256 * 2;  // Exceeding a block size.
```

Comment:
Hmm.  Interesting issue here.  This code can't know the per-block size that row_bit_count uses so there's no easily-accessible constant.  Maybe just crank this up to something unreasonable for any kind of block size?  Like 2048 or something.

Reply:
I started out trying to query the device for the maximum number of threads-per-block, and then got distracted.

> Maybe just crank this up to something unreasonable for any kind of block size?

Agreed. That would be a better bet. We're running a 1D kernel, and as per [the CUDA spec](https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#thread-hierarchy), we can't have more than 1024 threads per block. It'd be safest to use 2048 or some such.

Thanks for pointing this out. I'll fix this shortly.